qt/widgets/mplcpp/src/Zoomer.cpp

```diff
@@ -34,7 +34,9 @@ Python::Object mplNavigationToolbar(FigureCanvasQt *canvas) {
  * @param canvas A reference to an existing FigureCanvasQt object
  */
 Zoomer::Zoomer(FigureCanvasQt *canvas)
-    : Python::InstanceHolder(mplNavigationToolbar(canvas)), m_canvas(canvas) {}
+    : Python::InstanceHolder(mplNavigationToolbar(canvas)), m_canvas(canvas) {
+  pyobj().attr("hide")();
```

Comment:
Do you think it would be better to do this in the `mplNavigationToolbar` function, to avoid an additional retrieval of the python object? I can't see other calls of `mplNavigationToolbar`, so this shouldn't impact anything else. Something like:
```cpp
  auto obj = Python::Object(backend.attr(TOOLBAR_CLS)(
      canvas->pyobj(), canvas->pyobj(), showCoordinates));
  obj.attr("hide")();
```

Reply:
Good point. That function is entirely private so it makes sense. :+1:

Reply:
Done!